<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/stereo/depth/camera_info"/>
    <remap from="image_rect" to="/stereo/right/image"/>
    <remap from="points" to="/stereo/depth/points"/>
  </node>
</launch>